src/docs/cookbook/design/themes.md

```diff
@@ -10,36 +10,37 @@ next:
   path: /docs/cookbook/design/fonts
 ---
 
-In order to share colors and font styles throughout our app, we can take
-advantage of themes. There are two ways to define themes: App-wide or using
-`Theme` Widgets that define the colors and font styles for a particular part of
-our application. In fact, app-wide themes are just `Theme` Widgets created at
-the root of our apps by the `MaterialApp`!
-
-After we define a Theme, we can use it within our own Widgets. In addition, the
-Material Widgets provided by Flutter will use our Theme to set the background
+To share colors and font styles throughout an app, take
```

Comment:
take advantage of -> use

...to avoid the top (!) definition: "to be unfair". I forgot who pointed this out to me many years ago, but I try to avoid the phrase "take advantage of" now.